client/app/tasks/components/TaskDetailsModal.tsx

```diff
@@ -0,0 +1,131 @@
+import { useState } from 'react'
+import { useRouter } from 'next/navigation'
+import { format } from 'date-fns'
+import { FiEdit, FiX, FiTrash } from 'react-icons/fi'
+import { MdAccountCircle } from 'react-icons/md'
+import {
+  Dialog,
+  DialogContent,
+  Drawer,
+  DrawerContent,
+  Button,
+} from '@/_components/ui'
+import { colorMap } from '@/_consts'
+import { useMediaQuery } from '@/_hooks'
+import { TaskTypeWithUser } from '@/_types'
+import { DeleteTaskModal } from './DeleteTaskModal'
+
+type TaskDetailsModalProps = {
+  selectedTask: TaskTypeWithUser | null
+  isModalOpen: boolean
+  onClose: () => void
+  onOpen: () => void
+}
+
+const TaskDetailsModal = ({
+  selectedTask,
+  isModalOpen,
+  onClose,
+  onOpen,
+}: TaskDetailsModalProps) => {
+  const router = useRouter()
+  const isDesktop = useMediaQuery('(min-width: 768px)')
+  const [isDeleteModalOpen, setIsDeleteModalOpen] = useState(false)
+
+  if (!selectedTask) return null
+
+  const formattedDate = format(selectedTask.date, 'MMM/dd/yyyy')
+
+  const handleDeleteIcon = () => {
+    onClose()
```

Comment:
You've got the same problem I had before, which is showing the scrollbar with the modal

Reply:
Thanks! I fixed it with same way you did
https://github.com/HAK2024/fairy-share/pull/114/commits/01e827543c57dfcd337f48dcfb01a7d7d889d858